### ISSUE TITLE
How can l make the graph split into all possible cases l earned

### ISSUE THREAD

**Issue by samy12r** [samy12r/pcam-deeplearning#1]
I tried optimizer, batch normalization, and Xavier with n_layer and hid_dim adjusted values and got 12 values and I want to get a loss graph and train-val-acuity graph in 12 cases respectively, but the items are split into optimizers only and not into others. Anyone know a solution?

https://colab.research.google.com/drive/1kKdlhzfvF-zZAtGzV00z3M_nsW0gV17e#scrollTo=hPk5jQl2Bj-E&line=1&uniqifier=1
